mantidproject/mantid
# Indirect - JumpFit - Ensure output workspace is created and plotted in the interface

Currently, in the Interfaces > Indirect > Data Analysis > JumpFit interface, running the routine from this interface, does not create an output workspace, which results in an error being thrown when attempting to plot. This occurs as a result of the output workspace property not being set to the correct value when creating the fit algorithm.

This PR fixes this issue, by ensuring the workspace is created.

**To test:**
1. Navigate to Interfaces > Indirect > Data Analysis > JumpFit.
2. Provide the supplied test file in the 'Sample' field.
3. Select any fit function.
4. Click 'Run'.
5. Ensure the '*_Workspace', '*_Parameters' are created and the fit and difference are added to the preview plots. Also ensure the fitting parameters are updated in the property table.



Fixes #21204 . 

**Test Files**
[irs26176_graphite002_conv_1LFixF_s0_to_9_Result.zip](https://github.com/mantidproject/mantid/files/1518441/irs26176_graphite002_conv_1LFixF_s0_to_9_Result.zip)

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
